docs/codeql/codeql-for-visual-studio-code/troubleshooting-codeql-for-visual-studio-code.rst

```diff
@@ -38,6 +38,8 @@ You are most likely to need to restart the query server if you make external cha
 To see the logs from running a particular query, right-click the query in the Query History and select **Show Query Log**.
 If the log file is too large for the extension to open in the VS Code editor, the file will be displayed in your file explorer so you can open it with an external program.
 
+By default, the extension deletes logs after each workspace session. To override this behavior, you can specify a custom directory for query server logs. For more information, see ":ref:`Customizing settings <configuring-settings-for-running-queries>`."
```

Comment:
In the other article, you capitalized "Query Server" in Query Server logs. Should we do the same here, or not use capital letters at all?
Also, the link takes you to the main page ("Customizing settings"), can you link to the appropriate anchor (_configuring-settings-for-running-queries:?) or isn't that possible using Sphinx? Perhaps that's not what's done throughout these docs, this was just a thought.

Reply:
> In the other article, you capitalized "Query Server" in Query Server logs. Should we do the same here, or not use capital letters at all?

Good point. The extension itself uses title case, so I've tried to use "Query Server" when directly referring to a command or UI element, and just "query server" when referring to the query server itself. It's still fairly inconsistent, so I'll try to make this clearer!

> Also, the link takes you to the main page ("Customizing settings"), can you link to the appropriate anchor (_configuring-settings-for-running-queries:?) or isn't that possible using Sphinx? Perhaps that's not what's done throughout these docs, this was just a thought.

This should link to the anchor (and it does in a local Sphinx-generated preview). I think this is an issue with the HTML preview, since this syntax works correctly on the live site 🤔